pcweb/pages/docs/custom_components.py

```diff
@@ -186,7 +187,7 @@ def add_item(category: dict) -> rx.Component:
             updated_on_pypi_if_present(category["updated_on_pypi"]),
```

Comment:
How come this doesn't show up for any of them?

Reply:
because the CP doesn't expose them. I'm releasing a new change from CP in a bit.